editor/src/components/canvas/canvas-strategies/strategies/set-padding-strategy.spec.browser2.tsx

```diff
@@ -234,7 +234,7 @@ describe('Padding resize strategy', () => {
     await testPaddingResizeForEdge(editor, dragDelta, 'top')
     await editor.getDispatchFollowUpActionsFinished()
     expect(getPrintedUiJsCode(editor.getEditorState())).toEqual(
-      makeTestProjectCodeWithStringPaddingValues('8.2em 1em 3em 2em'),
+      makeTestProjectCodeWithStringPaddingValues('8.3em 1em 3em 2em'),
```

Comment:
what's this sneaky change? :D

Reply:
the way fractional values were rounded was changed due to one of Rheese's suggestions, and it showed up here (https://github.com/concrete-utopia/utopia/pull/2755#discussion_r1015685790)